lib/main_containers.pm

```diff
@@ -309,7 +309,7 @@ sub load_container_tests {
             loadtest 'containers/runc_integration' if (is_tumbleweed || is_sle || is_leap);
         }
         if (!check_var('NETAVARK_BATS_SKIP', 'all')) {
-            loadtest 'containers/netavark_integration' if (is_x86_64 && (is_tumbleweed || is_sle || is_leap));
+            loadtest 'containers/netavark_integration' if (is_tumbleweed || is_sle || is_leap);
```

Comment:
Careful, this would also enable it on all other architectures!

```suggestion
            loadtest 'containers/netavark_integration' if ((is_x86_64 || is_aarch64) && (is_tumbleweed || is_sle || is_leap));
```

Reply:
Not needed, because the integration tests need to be explicitly enabled (See line 301)